src/cli/backup/groups.go

```diff
@@ -0,0 +1,187 @@
+package backup
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/alcionai/corso/src/cli/flags"
+	"github.com/alcionai/corso/src/cli/utils"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+// ------------------------------------------------------------------------------------------------
+// setup and globals
+// ------------------------------------------------------------------------------------------------
+
+const (
+	groupsServiceCommand                 = "groups"
+	groupsServiceCommandCreateUseSuffix  = "--group <groupsName> | '" + flags.Wildcard + "'"
+	groupsServiceCommandDeleteUseSuffix  = "--backup <backupId>"
+	groupsServiceCommandDetailsUseSuffix = "--backup <backupId>"
+)
+
+// TODO: correct examples
```

Comment:
I think we can put empty examples, or maybe a warning message in here instead of a wrong example. Not that anyone should be seeing this, but just in case.